client-src/overlay.js

```diff
@@ -23,10 +23,26 @@ let iframeContainerElement;
 let containerElement;
 /** @type {Array<(element: HTMLDivElement) => void>} */
 let onLoadQueue = [];
+/** @type {any} */
```

Comment:
There are TS type definitions located in https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/trusted-types

However, I am not sure if it is worth typing so I've added `any` for now.

Reply:
I think we need install them in dev depedencies, because when types is not found it is `any`

Reply:
Fixed https://github.com/webpack/webpack-dev-server/pull/4404/commits/68d39742f2c13dc3beb68913eecee0170f7a69af